### PR TITLE
add dozzle service

### PR DESCRIPTION
**Pull request by Glup3**
# Adding new service Dozzle

> Dozzle is a small lightweight application with a web based interface to monitor Docker logs. It doesn’t store any log files. It is for live monitoring of your container logs only.

https://github.com/amir20/dozzle

I got the svg image from the repo under assets: https://github.com/amir20/dozzle/blob/master/assets/favicon.svg

### CODE REVIEW

**Comment by Glup3** [comment]
I haven't tested it yet - I need to setup my local environment first or is there an easier way to test new services?

**Comment by maietta** [comment]
> I haven't tested it yet - I need to setup my local environment first or is there an easier way to test new services?

There's a CONTRIBUTE.md in the project that details how to start the application for local dev.

**Comment by Glup3** [comment]
tested it on my machine & even found a bug. added new commit and it works now ✅

File: templates/compose/dozzle.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+services:
+  dozzle:
+    image: amir20/dozzle:latest
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://127.0.0.1:8080"]
+      interval: 2s
+      timeout: 10s
+      retries: 15
```
Comment (marcomaiermm) [inline templates/compose/dozzle.yaml:16]: How do you access the service? I would suggest to generate a FQDN.

```suggestion
services:
  dozzle:
    image: amir20/dozzle:latest
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      - SERVICE_FQDN_DOZZLE_8080
    healthcheck:
      test: ["CMD", "curl", "-f", "http://127.0.0.1:8080"]
      interval: 2s
      timeout: 10s
      retries: 15
```

Comment (Glup3) [reply templates/compose/dozzle.yaml]: ahhh, i missed that - good idea! thanks

**Review by marcomaiermm — APPROVED** [review]

**Comment by peaklabs-dev** [comment]
Thanks for your PR. Dozzle was added via this PR https://github.com/coollabsio/coolify/pull/3694 and fixed for the next release, so this is no longer needed.

**Pull request closed without merging**